AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTests.m

```diff
@@ -329,6 +365,45 @@ - (void)testTrackEventWithoutProperties {
   assertThat(name, is(expectedName));
 }
 
+- (void)testTrackEventWhenAnalyticsDisabled {
+
+  // If
+  id analyticsMock = OCMPartialMock([MSAnalytics sharedInstance]);
+  id logManagerMock = OCMProtocolMock(@protocol(MSLogManager));
+  OCMStub([analyticsMock isEnabled]).andReturn(NO);
+  [MSAppCenter configureWithAppSecret:kMSTestAppSecret];
+  [[MSAnalytics sharedInstance] startWithLogManager:logManagerMock appSecret:kMSTestAppSecret];
+
+  // FIXME: logManager holds session tracker somehow and it causes other test failures. Stop it for hack.
+  [[MSAnalytics sharedInstance].sessionTracker stop];
+
+  // When
+  OCMReject([logManagerMock processLog:OCMOCK_ANY forGroupId:OCMOCK_ANY]);
+  [[MSAnalytics sharedInstance] trackEvent:@"Some event" withProperties:nil];
```

Comment:
You can also verify `validateEventName:forLogType:` and `validateProperties:forLogName:andType:`  not being called.

Reply:
Same for trackPage test